scss/abstracts/_mixins.scss

```diff
@@ -0,0 +1,40 @@
+@use './colors' as color;
```

Comment:
I think this isn't necessary as those files are forwarded together?

Reply:
Yes, that's true. I'll get rid of that